### DISCUSSION TITLE
Disable illumination in contact sensor.

### DISCUSSION THREAD

**Discussion by dimkagithub** [merdok/homebridge-miot#462] [Q&A] [answered]
Hi!
After last update 1.6, contact sensor (isa.magnet.dw2hl) started to show illumination. It is ok but it can show only 2 states — 1 - weak, 2 - Strong. It is not luxes therefore it is pointless to use illumination in homekit.
How can I disable illumination?

<img width="1324" alt="Снимок экрана 2023-03-31 в 12 15 35" src="https://user-images.githubusercontent.com/71010307/229050228-548a6776-408d-4e85-a863-5aac8c4c05c2.png">

**Comment by merdok** [comment, marked as answer]
Well that is a surprise and makes sense...
You can use the suppressAutoServiceCreation property to disable it. Just check it out in the README.
